### PR TITLE
[PRODSEC-9364] added java-uuid-generator

### PR DESCRIPTION
**Pull request by code4uuuu**
https://hyland.atlassian.net/browse/PRODSEC-9364

### CODE REVIEW

File: repository/src/main/java/org/alfresco/repo/security/person/SplitPersonCleanupBootstrapBean.java [hunk 1]
```diff
@@ -1,32 +1,33 @@
...
+ * Alfresco is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ * 
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
+ * #L%
+ */
 package org.alfresco.repo.security.person;
```
Comment (suneet-gupta) [inline repository/src/main/java/org/alfresco/repo/security/person/SplitPersonCleanupBootstrapBean.java:26]: Why are we updating the header ?

File: repository/src/main/java/org/alfresco/repo/security/person/SplitPersonCleanupBootstrapBean.java [hunk 2]
```diff
@@ -1,32 +1,33 @@
...
+ * 
+ * Alfresco is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ * 
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
+ * #L%
+ */
```
Comment (suneet-gupta) [inline repository/src/main/java/org/alfresco/repo/security/person/SplitPersonCleanupBootstrapBean.java:25]: Why are we updating the header ?

File: repository/src/main/java/org/alfresco/repo/security/person/SplitPersonCleanupBootstrapBean.java [hunk 3]
```diff
@@ -1,32 +1,33 @@
-/*
- * #%L
- * Alfresco Repository
- * %%
- * Copyright (C) 2005 - 2016 Alfresco Software Limited
```
Comment (suneet-gupta) [inline repository/src/main/java/org/alfresco/repo/security/person/SplitPersonCleanupBootstrapBean.java:5]: Please change the year from 2016 to 2024.

**Review by suneet-gupta — APPROVED** [review]
LGTM

File: repository/src/main/java/org/alfresco/repo/security/person/SplitPersonCleanupBootstrapBean.java (hunk 3 above)
Comment (code4uuuu) [reply repository/src/main/java/org/alfresco/repo/security/person/SplitPersonCleanupBootstrapBean.java:5]: Done

**Review by purusothaman-mm — APPROVED** [review]
LGTM

**Review by SathishK-T — APPROVED** [review]

**Pull request merged**